pallets/simple_democracy/src/lib.rs

```diff
@@ -0,0 +1,334 @@
+//! This is a facade over Substrate's democracy pallet offering similar functionality with a few differences.
+//! This pallet implements a "simple majority" governance where only Council can vote but anyone can
+//! propose by locking a fixed amount of tokens, `PublicProposalDeposit`. Similar to Substrate's governance pallet, both Council and
+//! general public take turns proposing, and a proposal follows "propose" -> "referendum" -> "enact" sequence.
+//! General public's proposals can be "seconded" by other token holder.
+//! Technical committee can fast track proposal made by Council member.
+//! A simple majority of Council members' votes (>50%) are needed to accept an ongoing referendum.
+//! Council members can cancel a proposal made by the public.
+//! Setting up of Council and Technical committee can be seen in the pallet's tests.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+
+extern crate alloc;
+use alloc::vec::Vec;
+use frame_support::dispatch::{DispatchError, DispatchResult};
+use frame_support::{
+    decl_error, decl_event, decl_module, ensure,
+    traits::{Currency, EnsureOrigin, Get, Imbalance, OnUnbalanced},
+    weights::{DispatchClass, Weight},
+    StorageMap, StorageValue,
+};
+use frame_system::{self as system, ensure_signed};
+use pallet_democracy::{
+    AccountVote, BalanceOf, NegativeImbalanceOf, PreimageStatus, PropIndex, ReferendumIndex,
+    ReferendumInfo, ReferendumInfoOf, ReferendumStatus, Tally, UnvoteScope, VoteThreshold, Voting,
+};
+use sp_runtime::SaturatedConversion;
+
+#[cfg(test)]
+mod tests;
+
+// Another way to achieve voting by Council is to have the council vote as simple majority on the proposal as the gov. call.
+// The simple majority origin can be used to make the call.
+
+pub trait Trait: system::Trait + pallet_democracy::Trait + poa::Trait {
+    type Event: From<Event> + Into<<Self as system::Trait>::Event>;
+    type PublicProposalDeposit: Get<u64>;
+    /// Origin which can vote
+    type VoterOrigin: EnsureOrigin<Self::Origin, Success = Self::AccountId>;
+}
+
+decl_event!(
+    pub enum Event {
+        CouncilMemberAdded,
+    }
+);
+
+decl_error! {
+    pub enum Error for Module<T: Trait> {
+        OnlyDirectVotingAllowed,
+        OnlyStandardVotingAllowed,
+        NotVoter,
+        NoPermission,
+    }
+}
+
+decl_module! {
+    pub struct Module<T: Trait> for enum Call where origin: T::Origin {
+        type Error = Error<T>;
+
+        fn deposit_event() = default;
+
+        /// Almost similar to forked democracy pallet's `propose` with the difference being that the
+        /// deposit cannot be chosen by user but is fixed in `PublicProposalDeposit`
+        // TODO: Fix weight
+        #[weight = 0]
+        fn propose(origin, proposal_hash: T::Hash) {
+            let deposit = T::PublicProposalDeposit::get();
```

Comment:
`PublicProposalDeposit` is gameable by using multiple accounts. Is there a reason the feature is needed? If not, I suggest removing it.

Reply:
Removed the restriction and made it same as default democracy pallet.